### DISCUSSION TITLE
Media/video timestamp doubt

### DISCUSSION THREAD

**Discussion by FerroEduardo** [TeleMidia/ginga#181] [General]
I'm trying to build an application for my university discipline which simulates a multi-cam environment, like a spectator. The script will have access to some videos and I would like to change between them and keeping it synchronized (same timestamp).

Is it possible to do this with NCLua? I was trying to get a current video timestamp to set to another, but couldn't find any information about it.

**Comment by danielsm** [comment]
Hello @FerroEduardo,
In an NCLua script you cannot directly manipulate the content of a media, you only have access to its anchors and properties. You can change property values, as these are related to the way the media is displayed and not the content itself.

In the newest version of the NCL language, from the ABNT NBR 15606-2:2023 Standard, there is a currentTime property of continuous media (audio and/or video) that you could use to do what you suggested. However, our current reference implementation does not yet include this functionality.

With the current version of Ginga implemented, a simple solution for your application to keep multiple videos synchronized is to start them together, but controlling which video is selected with the properties visible and/or soundLevel.

**Comment by FerroEduardo** [comment]
> Hello @FerroEduardo,
> In an NCLua script you cannot directly manipulate the content of a media, you only have access to its anchors and properties. You can change property values, as these are related to the way the media is displayed and not the content itself.
> 
> In the newest version of the NCL language, from the ABNT NBR 15606-2:2023 Standard, there is a currentTime property of continuous media (audio and/or video) that you could use to do what you suggested. However, our current reference implementation does not yet include this functionality.
> 
> With the current version of Ginga implemented, a simple solution for your application to keep multiple videos synchronized is to start them together, but controlling which video is selected with the properties visible and/or soundLevel.
> 

Ok, thanks for the help! :)
